app/views/shared/transactions/_search.html.haml

```diff
@@ -2,7 +2,10 @@
   .row
     %fieldset.span6#search
       - @search.options.reject(&:multipart?).each do |searcher|
-        = f.input searcher.key, as: :transaction_chosen, collection: searcher.options, label: searcher.label, label_method: searcher.label_method, data_attrs: searcher.method(:data_attrs), input_html: { id: searcher.key }
+        - if searcher.key == "cross_cores"
+          = f.input searcher.key, collection: searcher.options, label: searcher.label, label_method: searcher.label_method, value_method: -> (option) { searcher.value_method(option) }, include_blank: false
```

Comment:
I went for an option that is the closest to what was already here, although I don't like this solution, so I'm up for a refactor (happy to hear opinions)